test/OpenTelemetry.Instrumentation.Http.Tests/HttpWebRequestActivitySourceTests.netfx.cs

```diff
@@ -413,8 +413,10 @@ public async Task TestTraceStateAndBaggage()
         [Theory]
         [InlineData("GET")]
         [InlineData("POST")]
-        public async Task DoNotInjectTraceParentWhenPresent(string method)
+        public async Task OverwriteTraceParentWhenPresent(string method)
```

Comment:
name is misleading.. as there is no validation done inside this test to validate if the instrumentation actually overwrites the header

Reply:
Renamed the test